src/types.ts

```diff
@@ -0,0 +1,44 @@
+import { Domain } from "@zero-tech/data-store-core";
+
+export interface DomainsResponse {
```

Comment:
I'm wondering if we should discard the DomainResponse type and keep our Domain API response object generic. You could coerce the response from getDomainById into a `DomainsResponse` instead.   

This way, when a user makes a call, they can always expect their response data to be inside `results`

Reply:
I wanted to keep numresults out of the single domain response, and also to keep the payload as same as the previous version

I can change it to return a domain object, to keep it simple, if that works for you

Reply:
My thinking was that nothing is consuming our previous version yet, and this would unify our API response object, making it a little easier to work with for the user.  

This is a nit though, if you prefer to leave it as is, I'm okay with that.

Reply:
Everyone has a different idea of how this payload should be structured, its on its 3rd modification now, maybe it would be best to tech design this